WordPressSharedTests/NSDateHelperTest.swift

```diff
@@ -33,22 +33,22 @@ class NSDateHelperTest: XCTestCase {
         XCTAssertEqual(components.month, data.month)
         XCTAssertEqual(components.day, data.day)
     }
-    
+
     /// Verifies that `mediumString` produces relative format strings when less than 7 days have elapsed.
-    func testMediumStringRelativeString() {
+    func testToMediumStringRelativeString() {
         let date = Date()
-        XCTAssertEqual(date.mediumString(), "just now")
-        XCTAssertEqual(date.addingTimeInterval(-60*5).mediumString(), "5 minutes ago")
-        XCTAssertEqual(date.addingTimeInterval(-60*60*2).mediumString(), "2 hours ago")
-        XCTAssertEqual(date.addingTimeInterval(-60*60*24).mediumString(), "1 day ago")
-        XCTAssertEqual(date.addingTimeInterval(-60*60*24*6).mediumString(), "6 days ago")
+        XCTAssertEqual(date.toMediumString(), "now")
+        XCTAssertEqual(date.addingTimeInterval(-60*5).toMediumString(), "5 minutes ago")
+        XCTAssertEqual(date.addingTimeInterval(-60*60*2).toMediumString(), "2 hours ago")
+        XCTAssertEqual(date.addingTimeInterval(-60*60*24).toMediumString(), "yesterday")
+        XCTAssertEqual(date.addingTimeInterval(-60*60*24*6).toMediumString(), "6 days ago")
     }
-    
+
     /// Verifies that  `mediumStringWithTime` takes into account the time zone adjustment
     func testMediumStringTimeZoneAdjust() {
         let date = Date()
         let timeZone = TimeZone(secondsFromGMT: Calendar.current.timeZone.secondsFromGMT() - (60 * 60))
-        XCTAssertEqual(date.mediumString(timeZone: timeZone), "just now")
+        XCTAssertEqual(date.toMediumString(inTimeZone: timeZone), "now")
```

Comment:
Should we stub the current locale in those tests? So that even if the simulator (or the system the tests are run on) is set to use a different locale than US English, the tests will still pass?

(same remark for all the tests in there, not just this one, obv)

Reply:
Yeah, no objection here – addressed in e9b84e354649c613d6b33067292e5b3bc76607a1.

Reply:
That commit fixes the timezone, not the locale, though 🙃

Reply:
Ummm yes, that is correct. Addressed in 9bb8610bf7f6f4ba6a3fb70e23828337bfe7f601, which converts the project to use a Test Plan, guaranteeing the locale. The time zone change is still useful, because we want to do the tests in UTC.